src/Features/VisualBasic/Portable/CodeRefactorings/InvertIf/InvertIfCodeRefactoringProvider.SingleLine.vb

```diff
@@ -0,0 +1,56 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.InvertIf
+    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeRefactoringProviderNames.InvertIf), [Shared]>
+    Friend NotInheritable Class VisualBasicInvertSingleLineIfCodeRefactoringProvider
```

Comment:
really nice that you separated out how MultiLine/SingleLine if's are handled.

Reply:
This was your idea a few month ago :)

Reply:
Lol...  Well.. I approve of younger Cyrus' idea :-)